lib/controllers/zodiac.js

```diff
@@ -0,0 +1,15 @@
+const { Router } = require('express');
+const { zodiac } = require('../zodiac-data');
+
+module.exports = Router()
+  .get('/:id', (req, res) => {
+    const match = zodiac.find((sign) => sign.id === req.params.id);
+    return res.json(match);
+  })
+  .get('/', (req, res) => {
+    const filteredData = [];
+    for (const sign of zodiac) {
+      filteredData.push({ id: sign.id, name: sign.name });
+    }
+    res.json(filteredData);
```

Comment:
Would prefer to see the use of a `.map()` here since it is purpose-built to return an array of equal length to your starting array